src/BoilerPlate/Player.cpp

```diff
@@ -9,75 +9,93 @@
 
 std::vector <Vector2> shipPoints, thrusterPoints;
 
+const float ACC_FORCE = 10.0f;
+const float MAX_SPEED = 1000.0f;
+const float OFFSET_ANG = 0.5f*MathUtilities::PI;
+const float DRAG_FACTOR = 0.98f;
+const float ROT_FACTOR = 7.0f;
+const int CHARGE_TIME = 10;
 
 Player::Player(int width, int height)
 	: SpaceObject(width, height), isThrusting(false) 
 {
-	position = new Vector2(Vector2::Origin);
-
-	shipPoints.push_back({ 0.0f, 33.0f });
-	shipPoints.push_back({ 30.0f, -27.0f });
-	shipPoints.push_back({ 12.0f, -17.0f });
-	shipPoints.push_back({ -12.0f, -17.0f });
-	shipPoints.push_back({ -30.0f, -27.0f });
-
-	thrusterPoints.push_back({ 12.0f, -20.0f });
-	thrusterPoints.push_back({ 15.0f, -22.0f });
-	thrusterPoints.push_back({ 11.0f, -21.0f });
-	thrusterPoints.push_back({ 17.0f, -25.0f });
-	thrusterPoints.push_back({ 10.0f, -24.0f });
-	thrusterPoints.push_back({ 16.0f, -32.0f });
-	thrusterPoints.push_back({ 8.0f, -26.0f });
-	thrusterPoints.push_back({ 12.0f, -32.0f });
-	thrusterPoints.push_back({ 10.0f, -30.0f });
-	thrusterPoints.push_back({ 14.0f, -36.0f });
-	thrusterPoints.push_back({ 6.0f, -32.0f });
-	thrusterPoints.push_back({ 8.0f, -38.0f });
-	thrusterPoints.push_back({ 0.0f, -55.0f });
-	thrusterPoints.push_back({ -8.0f, -38.0f });
-	thrusterPoints.push_back({ -6.0f, -32.0f });
-	thrusterPoints.push_back({ -14.0f, -36.0f });
-	thrusterPoints.push_back({ -10.0f, -30.0f });
-	thrusterPoints.push_back({ -12.0f, -32.0f });
-	thrusterPoints.push_back({ -8.0f, -26.0f });
-	thrusterPoints.push_back({ -16.0f, -32.0f });
-	thrusterPoints.push_back({ -10.0f, -24.0f });
-	thrusterPoints.push_back({ -17.0f, -25.0f });
-	thrusterPoints.push_back({ -11.0f, -21.0f });
-	thrusterPoints.push_back({ -11.0f, -21.0f });
-	thrusterPoints.push_back({ -15.0f, -22.0f });
-	thrusterPoints.push_back({ -15.0f, -22.0f });
-	thrusterPoints.push_back({ -12.0f, -20.0f });
+	charge = CHARGE_TIME;
+	m_position = Vector2(Vector2::Origin);
+	m_rotAng = OFFSET_ANG;
+	m_radius = 20.0f;
+
+	/*INITIALIZING SHIP POINTS*/
+	{
+		shipPoints.push_back({ 25.0f, 0.0f });
+		shipPoints.push_back({ -15.0f, 20.0f });
+		shipPoints.push_back({ -5.0f, 7.0f });
+		shipPoints.push_back({ -5.0f, -7.0f });
+		shipPoints.push_back({ -15.0f, -20.0f });
+	}
+
+	/*INITIALIZING THRUSTER POINTS*/
+	{
+		thrusterPoints.push_back({ -10.0f , 5.0f });
```

Comment:
Don't like the grouping using curly braces. You'd mentioned an approach using files, maybe it's time to move these guys out and load them only once on game initialization? Again not a requirement,